src/ape_cache/base.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+from sqlalchemy.ext.declarative import as_declarative, declared_attr  # type: ignore
```

Comment:
We should be typing checking against sqlalchemy because it has published types here: https://pypi.org/project/types-SQLAlchemy/

So you can add this package to `setup.py` in the lint section and then we also won't need this `type: ignore` anymore.